sound/soc/amd/acp-config.c

```diff
@@ -130,4 +130,15 @@ struct snd_soc_acpi_mach snd_soc_acpi_amd_sof_machines[] = {
 };
 EXPORT_SYMBOL(snd_soc_acpi_amd_sof_machines);
 
+struct snd_soc_acpi_mach snd_soc_acpi_amd_rmb_sof_machines[] = {
+	{
+		.id = "AMDI1019",
+		.drv_name = "rmb-dsp",
+		.pdata = (void *)&acp_quirk_data,
+		.fw_filename = "sof-rmb.ri",
```

Comment:
oh wow, we're not aligned between Intel and AMD.

we've removed the sof_fw_filename, and are using the default_fw_filename, see https://github.com/thesofproject/linux/pull/3658/commits/aaeba0996b29de0a382d5db4fc1cbe5c480844aa#diff-e13e023ae76311488644786679ee46d02aac2029c6739c83c3b0ae36c0c14825R74

Reply:
This file is not for SOF-linux.FIle rebased